templates/bashrc.osh-template

```diff
@@ -9,7 +9,10 @@ export OSH=~/.oh-my-bash
 
 # Set name of the theme to load. Optionally, if you set this to "random"
 # it'll load a random theme each time that oh-my-bash is loaded.
-OSH_THEME="font"
+OSH_THEME="random"
```

Comment:
Don't change the default theme.

Reply:
Sorry I checked in my local test! Will fix all these in my next commit.